thorlcr/master/thdemonserver.cpp

```diff
@@ -180,6 +200,12 @@ class DeMonServer : public CSimpleInterface, implements IDeMonServer
     {
         lastReport = msTick();
         reportRate = globals->getPropInt("@watchdogProgressInterval", 30);
+        IPropertyTree *costs = queryCostsConfiguration();
+        if (costs)
+        {
+            thorMasterRate = costs->getPropReal("thor/@master", 0.0);
+            thorSlaveRate = costs->getPropReal("thor/@slave", 0.0);
```

Comment:
minor: The other getPropX methods have defaults, I notice the new getPropReal doesn't.
Should probably be changed to be consistent, and would not need 0.0 passed here.